lib/src/screen/dice/dice_screen.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class DiceScreen extends StatefulWidget {
+  DiceScreen({Key key, this.title})
+      : super(key: key);
+
+  final String title;
+
+
+  @override
+  _DiceScreenState createState() => _DiceScreenState();
+
+}
+
+class _DiceScreenState extends State<DiceScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(widget.title),
+      ),
+      body: Center(
+        child: StreamBuilder(
+          initialData: 0,
+          builder: (BuildContext context, AsyncSnapshot<int> snapshot) {
+            return Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                Text(
+                  'Roll a dice:',
+                ),
+                Text(
+                  '${snapshot.data}',
+                  style: Theme.of(context).textTheme.headline4,
+                ),
+              ],
+            );
+          },
+        ),
+        // child:
+      ),
+      floatingActionButton: Row(
+        mainAxisAlignment: MainAxisAlignment.end,
+        children: [
+          FloatingActionButton(
+            backgroundColor: Colors.green,
+            child: Icon(Icons.autorenew_rounded),
+          ),
+          SizedBox(
+            width: 10.0,
```

Comment:
it is possible in Flutter to use a constant or something like that?
If so, please use a constant here to avoid the hardcode

Reply:
Good practice is to create a global constants file like:

```
abstract class DiceDimens {
  static const defaultDividerHeight = 10.0;
  static const defaultIconHeight = 14.0;
```

Reply:
I create a class (dice_dimens) that contains constants to replace in code. Thanks guys!